util/utils.go

```diff
@@ -9,6 +9,12 @@ import (
 
 type PrintCallback func(format string, args ...interface{})
 
+func init() {
+	if PrintUtil == nil {
+		InitPrinter(PrintErr)
+	}
+}
+
```

Comment:
There is code strewn about that initializes the printer with the preferred type of printer if it's currently nil.  I don't know if this will break that because this init method is automatically called and the printer will no longer be nil in those cases.

Reply:
Should I added the init to be local to `DockerPull`? I am not using the Printer in my stuff right now so I don't really want to import it just to initialize.